classes/output/mobile.php

```diff
@@ -186,6 +186,11 @@ public static function forum_discussions_view($args) {
                 $discussion->contribslabel = ($stats['contribs'] == 0) || ($stats['contribs'] > 1) ? get_string('contributors', 'hsuforum') : get_string('contributor', 'hsuforum');
                 $discussion->subscribedlabel = $discussion->subscriptionid ? get_string('toggled:subscribe', 'hsuforum') : get_string('toggle:subscribe', 'hsuforum');
                 $discussion->replylabel = ($discussion->replies == 0) || ($discussion->replies > 1) ? get_string('replies', 'hsuforum') : get_string('reply', 'hsuforum');
+                
+                // If we find the @@PLUGIN@@ in a message string we know we need to get the correct embedded images
+                if(strpos($discussion->message, '@@PLUGINFILE@@') !== false) {
+                    $discussion->message = self::returnEmbeddedImageMessage($discussion->message, $context->id, $discussion->id);
```

Comment:
Not sure why the returnEmbeddedImageMessage() is in the mobile.php We have a mobilelib for helper functions. Not sure if I am missing something.

Reply:
you're right thanks missed that